### PR TITLE
add url to 'Add News' link in footer

### LINKED ISSUE #258: "Add News" link broken
In the footer, the Add News link doesn't point to a URL. it should point to the page in the admin panel where users can create news articles.

### PR DESCRIPTION
**Pull request by jacobdgm**
fixes #258

### CODE REVIEW

File: django/cantusdb_project/templates/base.html [hunk 1]
```diff
@@ -325,7 +325,7 @@ <h5>Contributor Navigation</h5>
                                 {% endif %}
                                 {% if request.user|has_group:"project manager" %}
                                     <li class="list-group-item p-0">
-                                        <a href="">Add News</a>
+                                        <a href="/admin/articles/article/add/">Add News</a>
```
Comment (jwang44) [inline django/cantusdb_project/templates/base.html:328]: Is it possible to avoid hard-coding this url? Does `{% url ... %}` work?

Comment (jacobdgm) [reply django/cantusdb_project/templates/base.html:328]: I'm not certain, but I think that to link to the Admin panel, we have to hard-code it. I think if we had an article-create view, we could do it with `{% url ... %}`, but we don't so we can't.

Comment (jacobdgm) [reply django/cantusdb_project/templates/base.html:328]: this is the same format we use a couple of lines later, for "Add Feast", "Add Indexer", etc.

Comment (jwang44) [reply django/cantusdb_project/templates/base.html:328]: Can you look into this? https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#reversing-admin-urls
If possible, we should replace all hard-coded urls to Django admin interface with url tags.

Comment (jwang44) [reply django/cantusdb_project/templates/base.html:328]: Merging for now. Replacing the urls later

**Pull request merged**